locales/it/index.yml

```diff
@@ -2075,7 +2075,10 @@ bonus:
         rankTitle: "Classifica Super Cashback"
         howItWorks:
           title: "Come funziona?"
-          body: "I **{{citizens}} cittadini** con il più alto numero di transazioni **ricevono il Super Cashback, un premio pari a €{{amount}}.**\n\n**La classifica cambia ogni giorno.** Quella definitiva sarà visualizzata il {{endDate}}."
+          body: "I **{{citizens}} cittadini** con il più alto numero di transazioni **ricevono il Super Cashback, un premio pari a {{amount}}.**"
```

Comment:
@Undermaken What do you think, instead, if we format this from the utility function and not inside locales?

Reply:
It's pretty the same but since this text coming from locales, an editor can't know if that symbol is displayed without reading the code.
imho including it here could have an advantage